pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -380,6 +380,7 @@ class Build {
                         if ("${testType}".contains("external")) {
                             DOCKER_REGISTRY_URL = "sys-rt-docker-local.artifactory.swg-devops.com"
                             DOCKER_REGISTRY_URL_CREDENTIAL_ID = artifactoryCredential
+                            additionalTestLabel += "&&!sw.os.ubuntu.22"
```

Comment:
This is inside the foreach loop. Are we sure this works as intended? Ie. Once it is added, it doesn't get applied to other targets?

Reply:
An extra variable `curAdditionalTestLabel` has been added inside the loop to avoid affecting `additionalTestLabel`. Thanks.